config/jobs/kubernetes-sigs/sig-windows/sig-windows-config.yaml

```diff
@@ -381,7 +381,7 @@ presubmits:
     decorate: true
     decoration_config:
       timeout: 2h
-    always_run: false
+    always_run: true
```

Comment:
/hold
Please don't add new presubmits to always run in kubernetes/kubernetes without discussion.

Reply:
here is the original tracking issue: https://github.com/kubernetes/kubernetes/issues/97304, I changed the PR a little ro `run_if_changed: 'azure.*\.go$|.*windows\.go$'`
@BenTheElder

Reply:
is that ok to merge this PR since there is already discussion here: https://github.com/kubernetes/kubernetes/issues/97304
The azure file test on windows containerd is stable now, we want to make it if there is windows code change, thanks.

Reply:
We are behind on documenting policy for this, so here's my take as a SIG Testing chair.

Adding a new merge-blocking job needs careful consideration (ref: https://github.com/kubernetes/test-infra/issues/18729)
- Merge-blocking jobs are much more expensive to run given the volume of PR traffic we receive
- More jobs-per-PR means higher flakes-per-PR if we assume each job has some non-zero chance of flaking

We also want to ensure that merge-blocking coverage is a subset of release-blocking coverage.  That is:
- if there are tests that are not release-blocking, they should not be merge-blocking
- If there are ~(non-esoteric, non-expensive, high-signal) tests that are part of release-blocking, we should aspire to run them as part of merge-blocking to get earlier signal on whether changes are likely to break the release

The reason we allowed ourselves the extra jobs while switching from dockershim to containerd for the jobs mentioned in https://github.com/kubernetes/kubernetes/issues/97304 (gce, scalability) is that we were replacing one merge-blocking job for another.

tl;dr if there is release-blocking coverage, and the job cost isn't too egregious, then I'm probably ok with run_if_changed for now

Reply:
thanks for clarification, it's not blocking test since it's optional test.
/hold cancel